bots/create-bot-users/README.md

```diff
@@ -1,8 +1,8 @@
-# Creating Bot Users
+# Create Bot Users
 
 In order to talk to your chatbot there must be a bot user pre-configured on the Rocket.Chat server.
 
-## Creating a User on the Server
+## Create a User on the Server
 
 An admin user is required to create the account for the bot to login to.
```

Comment:
# Create a User on the Server

An admin user is required to create a special user account which the bot will use to login and listen for messages or send messages.

You must login to your Rocket.Chat server as an Admin user and then create a new user.

Reply:
This file should be deleted as redundant